src/steamship/plugin/service.py

```diff
@@ -103,15 +100,15 @@ def model_cls(self) -> Type[TrainableModel]:
 
     def run(self, request: PluginRequest[T]) -> Union[U, Response[U]]:
         """Loads the trainable model before passing the request to the `run_with_model` handler on the subclass."""
-        logging.info("TrainablePluginService:run() - Loading model")
+        self.logger.info("TrainablePluginService:run() - Loading model")
         model = self.model_cls().load_remote(
             client=self.client,  # This field comes from being a subclass of App
             plugin_instance_id=request.plugin_instance_id,
             checkpoint_handle=None,  # Will use default
             use_cache=True,
             plugin_instance_config=self.config,
         )
-        logging.info("TrainablePluginService:run() - Loaded model; invoking run_with_model")
+        self.logger.info("TrainablePluginService:run() - Loaded model; invoking run_with_model")
```

Comment:
Ah, I see -- so we'll encourage folks to use `self.logger`. Fine w me!

It would be cool if there were a way to hijack python's default logger, just to catch other logging messages that might be emitted by third party libraries, etc. Could investigate in a future pr...